### DISCUSSION TITLE
maturin new not recognized

### DISCUSSION THREAD

**Discussion by henceiusegentoo** [PyO3/maturin#1675] [Q&A] [answered]
```
[leander@DESKTOP-VRLDOLM testing]$ maturin new
error: unrecognized subcommand 'new'

Usage: maturin <COMMAND>

For more information, try '--help'.

leander@DESKTOP-VRLDOLM testing]$ maturin --version
maturin 1.1.0
```

I'm on Artix

**Comment by messense** [comment, marked as answer]
You mean you installed maturin from Artix package manager right? 

I think this means that they didn't compile maturin with full set of features, you can ask them to do so by setting `MATURIN_SETUP_ARGS=--features full,native-tls` env var before compiling.

**Reply by henceiusegentoo** [reply]
Yes, i did `sudo pacman -S python-maturin`. The same problem seems to exist with the arch package. I will inform them, thank you!

**Comment by henceiusegentoo** [comment]
I just wanna add: I fixed the issue by adding arch-repos to Artix, installing pipx and then installing maturin via pipx. Works like a charm now
